banyand/measure/merger.go

```diff
@@ -179,15 +183,21 @@ var reservedDiskSpace uint64
 
 func (tst *tsTable) getPartsToMerge(snapshot *snapshot, maxFanOut uint64) ([]*partWrapper, map[uint64]struct{}) {
 	var parts []*partWrapper
-	toBeMerged := make(map[uint64]struct{})
+
+	maxBytesPerPart := maxFanOut / 2
```

Comment:
We should parameterize it, taking free disk space into account if not set by the end user.